src/styles/global.scss

```diff
@@ -36,6 +36,8 @@ body {
   margin: 0;
   padding: 0;
   background-color: color('sky', 'light');
+  // scrollbar width varies across browsers and operating systems, this value needs to be considered when calculating horizontal alignment of overlays so it must be consistent
+  scrollbar-width: scrollbar-width();
```

Comment:
Length values were [dropped from the spec](https://github.com/w3c/csswg-drafts/commit/3359878276fba119ab68a7bc526c6f205cad289d), the only available values are now `auto | thin | none`.